File: baselibrary/src/main/java/com/dbvr/baselibrary/model/MainMo.java
package com.dbvr.baselibrary.model;

import java.io.Serializable;

/**
 * 直播短视频商品类型
 */
public class MainMo implements Serializable {
    private int id;
    private boolean isFollow;

    private String userId;
    private String nickName;
    private String headUrl;
    private String permanentResidence;//常住地
    private String jumpNo;//跳跳号
    private String fansNumber;//粉丝量
    private String  followNumber;//灌注量
    private String  goodsNumber;//商品数量

    private String roomId;
    private String liveTitle;
    private String coverUrl;
    private String lookNum;//观看人数
    private String praseCount;//点赞数量

    private String goodsTitle;
    private String goodsPrice;
    private String goodsCover;

    public MainMo() {
    }

    public boolean isFollow() {
        return isFollow;
    }

    public String getPermanentResidence() {
        return permanentResidence;
    }

    public void setPermanentResidence(String permanentResidence) {
        this.permanentResidence = permanentResidence;
    }

    public String getJumpNo() {
        return jumpNo;
    }

    public void setJumpNo(String jumpNo) {
        this.jumpNo = jumpNo;
    }

    public String getFansNumber() {
        return fansNumber;
    }

    public void setFansNumber(String fansNumber) {
        this.fansNumber = fansNumber;
    }

    public String getFollowNumber() {
        return followNumber;
    }

    public void setFollowNumber(String followNumber) {
        this.followNumber = followNumber;
    }

    public String getGoodsNumber() {
        return goodsNumber;
    }

    public void setGoodsNumber(String goodsNumber) {
        this.goodsNumber = goodsNumber;
    }

    public void setFollow(boolean follow) {
        isFollow = follow;
    }

    public String getRoomId() {
        return roomId;
    }

    public void setRoomId(String roomId) {
        this.roomId = roomId;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getNickName() {
        return nickName;
    }

    public void setNickName(String nickName) {
        this.nickName = nickName;
    }

    public String getHeadUrl() {
        return headUrl;
    }

    public void setHeadUrl(String headUrl) {
        this.headUrl = headUrl;
    }

    public String getLiveTitle() {
        return liveTitle;
    }

    public void setLiveTitle(String liveTitle) {
        this.liveTitle = liveTitle;
    }

    public String getCoverUrl() {
        return coverUrl;
    }

    public void setCoverUrl(String coverUrl) {
        this.coverUrl = coverUrl;
    }

    public String getLookNum() {
        return lookNum;
    }

    public void setLookNum(String lookNum) {
        this.lookNum = lookNum;
    }

    public String getPraseCount() {
        return praseCount;
    }

    public void setPraseCount(String praseCount) {
        this.praseCount = praseCount;
    }

    public String getGoodsTitle() {
        return goodsTitle;
    }

    public void setGoodsTitle(String goodsTitle) {
        this.goodsTitle = goodsTitle;
    }

    public String getGoodsPrice() {
        return goodsPrice;
    }

    public void setGoodsPrice(String goodsPrice) {
        this.goodsPrice = goodsPrice;
    }

    public String getGoodsCover() {
        return goodsCover;
    }

    public void setGoodsCover(String goodsCover) {
        this.goodsCover = goodsCover;
    }
}

File: baselibrary/src/main/java/com/dbvr/baselibrary/model/MainMo.kt
package com.dbvr.baselibrary.model

import java.io.Serializable

/**
 * 直播短视频商品类型
 */
class MainMo : Serializable {
    //公共属性
    var id: Int = 0//短视频或直播的ID
    var labelId : String ?= null//分类
    var live: Boolean = false//是否是直播
    var isFollow: Boolean = false//是否已关注
    var title: String? = null//直播/短视频 标题
    var coverUrl: String? = null//直播/短视频 封面
    var userId: String? = null//用户ID
    var nickName: String? = null//用户昵称
    var headUrl: String? = null//用户头像
    var goodsTitle: String? = null//商品标题
    var goodsPrice: String? = null//商品价钱
    var goodsCover: String? = null//商品封面

    //直播属性
    var permanentResidence: String? = null//常住地
    var jumpNo: String? = null//跳跳号
    var fansNumber: String? = null//粉丝量
    var followNumber: String? = null//关注量
    var goodsNumber: String? = null//商品数量
    var roomId: String? = null//房间id
    var lookNum: String? = null//观看人数

    //短视频属性
    var praseCount: String? = null//点赞数量
    var commentNum: String? = null//评论数量
    var videoUrl: String? = null//播放地址


}
